cms/djangoapps/contentstore/views/course.py

```diff
@@ -339,6 +339,39 @@ def _course_outline_json(request, course_module):
     )
 
 
+def get_in_process_course_actions(request):
+    """
+     Get all inprocess course actions
+    """
+    return [
+        course for course in
+        CourseRerunState.objects.find_all(
+            exclude_args={'state': CourseRerunUIStateManager.State.SUCCEEDED}, should_display=True
+        )
+        if has_studio_read_access(request.user, course.course_key)
+    ]
+
+
+def _staff_accessible_course_list(request):
+    """
+    List all courses available to the logged in user by iterating through all the courses
+    """
+    def course_filter(course):
+        """
+        Filter out unusable and inaccessible courses
+        """
+        # pylint: disable=fixme
+        # TODO remove this condition when templates purged from db
+        if course['location'].course == 'templates':
+            return False
+
+        return has_studio_read_access(request.user, course['locator'])
+
+    courses = filter(course_filter, modulestore().get_courses_summary())
```

Comment:
throughout this file, it would be clearer I think to try to refer to course summaries as `course_summaries` and `course_summary` instead of `course` and `courses`.